CHANGELOG.next.md

```diff
@@ -17,6 +17,7 @@ Thanks, you're awesome :-) -->
 #### Added
 
 * Add `orchestrator` fieldset to experimental schema. #1292
+* Extend `threat` fieldset to stage 1 RFC. #1335
```

Comment:
Please remove the changelog entry since we don't add changelog entry for the RFC PRs.

We'll add an entry as these proposed changes are implemented after the RFC advances.

Reply:
Done.